src/libp2p/chain_exchange/message.rs

```diff
@@ -38,6 +38,11 @@ impl ChainExchangeRequest {
     pub fn include_messages(&self) -> bool {
         self.options & MESSAGES > 0
     }
+
+    /// If the value of `options` is valid
```

Comment:
This comment is not very helpful. Could elaborate here?

Reply:
How about `/// If either the [HEADERS] bit or the [MESSAGES] bit is set.`?

Reply:
@LesnyRumcajs please re-approve if the change looks OK as it's required by the new branch rule.